### ISSUE TITLE
Missing tool on Mac - realpath

### ISSUE THREAD

**Issue by johnpmayer** [spatialos/gdk-for-unity-fps-starter-project#40]
#### Description

Setup script threw an unexpected warning

## Current behaviour

```
scripts/shell/setup.sh: line 23: realpath: command not found
This script will create the following directory: 
*** If such a directory already exists it will be deleted. ***
Are you ready to proceed? (y/n)
```

## Possible solution

Suggest adding the following to the instructions

```
brew install coreutils
```

## Environment

MacOS

**Comment by johnpmayer** [comment]
In case it wasn't clear, this happened when running

```
bash scripts/shell/setup.sh
```

**Comment by jamiebrynes7** [comment]
Hey @johnpmayer, thanks for the report! We've got a fix for this in #32. Need to do some testing before merging.

In the meantime, a workaround can be to install `coreutils` via `brew install coreutils` or to go through the manual setup guide.

**Comment by jamiebrynes7** [comment]
Ah I've seen you suggested the `coreutils` installation in the original report 😅

**Comment by mattyoung-improbable** [comment]
Fixed in 0.1.1
